### DISCUSSION TITLE
All processes in Wait_Start mode.

### DISCUSSION THREAD

**Discussion by qradar** [Security-Onion-Solutions/securityonion#6174] [Unsupported Versions] [answered]
I have installed Security Onion in Virtual Box. During installation after selecting Linux there was no option for CentOS so I selected Fedora and proceeded with the installation.

When the installation was at 95 percent it said something like Verifying Setup and then gave this error:

![Sec](https://user-images.githubusercontent.com/93916912/141094064-5672fc55-1f8e-45b5-a141-bab22dcadd12.JPG)

Now when I use sudo so-status command it shows me this:

 ![Sec1](https://user-images.githubusercontent.com/93916912/141094293-4f448319-0328-4ce8-a41f-0dd69fef75d3.JPG)

I have given 20GB RAM and 6 processors still services are not starting even if I keep Virtual Machine running for hours.

Please help :(

**Comment by dougburks** [comment, marked as answer]
Per the error message, have you checked `/root/errors.log` and `/root/sosetup.log` for additional clues?

**Reply by qradar** [reply]
Thanks for the reply. You can close this thread as my problem has been solved.
